### ISSUE TITLE
the data processing format and the metrics of event detection

### ISSUE THREAD

**Issue by Log4J-Ops** [THU-KEG/CLEVE#12]
I used your model but cannot reproduce the same performance. Can you provide the processing data format?
The calculation process of the metrics uses the command "from sklearn.metrics import f1_score, precision_score," tha library recognizes events one token for one trigger. However, the definition of event detection is to identify  and class the span of a trigger when the trigger consists of multi-tokens . Is it reasonable to use this library in this case?

**Comment by Bakser** [comment]
Regarding the data format, this repo adopts the same data format as in [HMEAE](https://github.com/thunlp/HMEAE), which shares the preprocessing script and format example.
Regarding the evaluation metric, please kindly note that the metrics are calculated over sequences of `trigger candidates` rather than `tokens`.
We recommend trying our recent [OmniEvent toolkit](https://github.com/THU-KEG/OmniEvent), which implements standardized data preprocessing and evaluation metrics. You can also easily customize your own models within its framework. CLEVE is also included in OmniEvent.
